onnxmltools/convert/common/optimizer.py

```diff
@@ -35,7 +35,8 @@ def is_transpose(self):
 
     @property
     def in_single_path(self):
-        return len(self.successor) <= 1
```

Comment:
Do you mind to add a doc string for this function?

Reply:
added